### PR TITLE
style: add prettier plugin to sort imports

### PR DESCRIPTION
**Pull request by Michael-A-Heuer**
## Description

This PR adds project-wide format checks for all files matching this pattern `'**/*.{js,json,md,sol,ts,yml}'`.

Task ID: [OS-594](https://aragonassociation.atlassian.net/browse/OS-594)

## Type of change

- formatting and style

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [x] I have updated the `CHANGELOG.md` file in the root folder.
- [x] I have updated the `DEPLOYMENT_CHECKLIST` file in the root folder.
- [x] I have updated the `UPDATE_CHECKLIST` file in the root folder.

[OS-594]: https://aragonassociation.atlassian.net/browse/OS-594?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,5 +18,14 @@
       "**/matchstick-as",
       "**/matchstick-as/**"
     ]
+  },
+  "scripts": {
```
Comment (Michael-A-Heuer) [inline package.json:22]: Likewise, we should move the Eslinting here (and actually make it work).

File: .github/workflows/style-check.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+name: Style Checks
+on:
+  push:
+
+env:
+  working-directory: .
```
Comment (mathewmeconry) [inline .github/workflows/style-check.yml:6]: working directory is not necessary when it is in the root of the project

**Review by mathewmeconry — CHANGES_REQUESTED** [review]

File: .github/workflows/style-check.yml (hunk 2 above)
Comment (Michael-A-Heuer) [reply .github/workflows/style-check.yml]: Fixed in e495f068835648b821bef8e10eeb23be6eb3408a

**Review by mathewmeconry — APPROVED** [review]

**Review by Rekard0 — APPROVED** [review]
Nice

**Pull request merged**